src/scheduler/fifo.c

```diff
@@ -1561,34 +1561,9 @@ run_update_resresv(status *policy, int pbs_sd, server_info *sinfo,
 			update_job_comment(pbs_sd, array, buf);
```

Comment:
I've kept it in scheduler since it is generic to all the subjobs in the array. We are updating this comment in server after receiving the reply from mom reflecting whether it has started/failed.